src/main/antora/modules/archive/pages/ent-beans/ent-beans.adoc

```diff
@@ -0,0 +1,3 @@
+= Jakarta Enterprise Beans (and Application Client Container)
+
+[TBD]
```

Comment:
Let's point to https://eclipse-ee4j.github.io/jakartaee-tutorial/#using-the-embedded-enterprise-bean-container for now.

Reply:
@BalusC do you mean the 9.1 version of this or the 10.0 version of this?  (this link points to the old docs).

Reply:
Just xref the intended adoc.